### PR TITLE
FIX : correct display of html notes in lists

### PR DESCRIPTION
**Pull request by altairis-tof**

### CODE REVIEW

File: htdocs/comm/propal/list.php [hunk 1]
```diff
@@ -2321,17 +2321,17 @@
 			}
 			// Note public
 			if (!empty($arrayfields['p.note_public']['checked'])) {
-				print '<td class="center">';
-				print dol_string_nohtmltag($obj->note_public);
+				print '<td class="sensiblehtmlcontent center">';
+				print dol_string_onlythesehtmltags($obj->note_public);
```
Comment (eldy) [inline htdocs/comm/propal/list.php:2325]: With v19, there is a new method
dolPrintHtml... 
Can you try with it ?

Comment (altairis-tof) [reply htdocs/comm/propal/list.php]: cool, i did it @eldy

**Comment by altairis-tof** [comment]
hello @eldy i have several PRs like this one with phpcs problems apparently not related to my work. 
what should i do in this case ?

**Comment by eldy** [comment]
> hello @eldy i have several PRs like this one with phpcs problems apparently not related to my work. 
> what should i do in this case ?

In such case, you must try this: redo a pull on the branch you did the development on. This will update branch the or us tested on and should solve the error that is inside the branch with date when pr was pushed. You may also try to repush on your fork.
In most cases i have a buttom on github on the PR to do it myself, button is called "Update branch". But for an unknown reason (if someone has answer, thanks to comment), i do not have it on some PR like this one and others you did. So only you can do it.

**Comment by altairis-tof** [comment]
salut @eldy, il reste une erreur bizarre qui ne vient pas de moi.
image

**Comment by mdeweerd** [comment]
> salut @eldy, il reste une erreur bizarre qui ne vient pas de moi. image

Il y a eu des évolutions dans les tests pour que cela passe sous windows, et ce problème devrait en faire parti.  Donc mettre à jour le PR.

Je pense aussi probable que @eldy ne peut pas faire "Update branch" si les modifications par les "maintainers" ne sont pas autorisés pour le PR.

**Comment by altairis-tof** [comment]
PR ready for merge @eldy

**Pull request merged**